src/babel-benchmark.js

```diff
@@ -2,14 +2,26 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-const Babel = require("@babel/standalone");
+const Babel = require("@babel/core");
 const babylon = require("babylon");
 const fs = require("fs");
 
 const payloads = [
   {
     name: "vue.runtime.esm-nobuble-2.4.4.js",
-    options: { presets: ["es2015"], sourceType: "module" }
+    options: {
+      presets: [
+        [
+          "@babel/preset-env",
+          {
+            targets: {
+              browsers: ["last 2 versions"]
+            }
```

Comment:
Please add current + lts versions of Node here as well.

Reply:
Done.